jwst/resample/tests/test_resample_step.py

```diff
@@ -235,3 +235,23 @@ def test_pixel_scale_ratio_imaging(nircam_rate, ratio):
     area1 = result1.meta.photometry.pixelarea_steradians
     area2 = result2.meta.photometry.pixelarea_steradians
     assert_allclose(area1 * ratio**2, area2, rtol=1e-6)
+
+
+def test_sip_coeffs_do_not_propagate(nircam_rate):
+    im = AssignWcsStep.call(nircam_rate, sip_degree=2)
```

Comment:
Btw, if I didn't specify `sip_degree=2` here, the sip fitting would fit with just a 1st order affine transformation, i.e. just a CD matrix, so no SIP coeffs and ctypes didn't have the `-SIP` suffix.